src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -310,16 +310,22 @@ class ArrayField extends React.Component {
           )}
           {title &&
             !itemCountLocked && (
-              <button
-                type="button"
-                disabled={addAnotherDisabled}
-                className="edit-btn primary-outline"
-                onClick={() => this.handleAdd()}
-              >
-                {uiOptions.itemName
-                  ? `Add Another ${uiOptions.itemName}`
-                  : 'Add Another'}
-              </button>
+              <>
+                <button
+                  type="button"
+                  disabled={addAnotherDisabled}
+                  className="edit-btn primary-outline"
+                  onClick={() => this.handleAdd()}
+                >
+                  {uiOptions.itemName
+                    ? `Add Another ${uiOptions.itemName}`
+                    : 'Add Another'}
+                </button>
+                <div>
+                  {addAnotherDisabled &&
```

Comment:
![screen shot 2019-03-06 at 10 01 34](https://user-images.githubusercontent.com/4998130/53899356-421a4280-3ff7-11e9-8b32-f4c25fb34b91.png)
![screen shot 2019-03-06 at 10 01 21](https://user-images.githubusercontent.com/4998130/53899358-434b6f80-3ff7-11e9-8b28-4fa846c47fcb.png)

Reply:
There was a requirement to use specific item name in the copy [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16659#issuecomment-466103264). AFAIK there's not a properly cased string to use for this. 

- itemName: singular and wouldn't work in this sentence e.g. service period
- title: first character is capitalized- can't reliably lower case this since the first word may be a proper noun e.g. Sponsor's service periods

Reply:
Yeah, I think we'd have to add a config option for this.